pygmt/tests/test_info.py

```diff
@@ -74,16 +75,30 @@ def test_info_2d_list():
     assert output == expected_output
 
 
-def test_info_series():
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        "int64",
+        pytest.param("int64[pyarrow]", marks=td.skip_if_no(package="pyarrow")),
```

Comment:
Using this `pandas.util._test_decorators.skip_if_no` pytest mark, following https://github.com/pandas-dev/pandas/blob/2a953cf80b77e4348bf50ed724f8abc0d814d9dd/pandas/tests/series/methods/test_tolist.py#L27-L29. Technically this is a private function, so happy to remove it too.

Reply:
Doesn't [`pytest.importorskip`](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip-ref) work here?

Reply:
No, if we use `pytest.impororskip` here, all the tests in this file would be skipped if `pyarrow` is not installed.

Reply:
Maybe we can try something like:
```python
try: 
    import pyarrow
    has_pyarrow = True
except ImportError:
    has_pyarrow = False

pytest.param("int64[pyarrow]", marks=pytest.mark.skipif(condition=not has_pyarrow)),
```

Reply:
Just tried it at commit 5adcba6e1c6f16304a0460e2c66af64a293c0dbf and it works, but the code is much longer. Maybe we should just reimplement this pytest mark from pandas under `pygmt/helpers/testing.py`, since it will be used a lot for #2800.

Reply:
Ok, opened a separate PR for adding the `skip_if_no` helper function at https://github.com/GenericMappingTools/pygmt/pull/2883.